src/pages/TeamSettings/KV.vue

```diff
@@ -0,0 +1,636 @@
+          no-data-text="Your team does not have any keys yet."
+          @item-expanded="handleKVExpand"
+        >
+          <!-- ACTIONS -->
+          <template #expanded-item="{ headers, item }">
+            <td :colspan="headers.length">
+              <div
+                class="d-flex justify-end align-end mt-5"
+                style="width: 100%;"
+              >
+                <v-btn
+                  x-small
+                  class="text-normal mr-2"
+                  depressed
+                  color="utilGrayLight"
+                  title="Reset"
+                  @click="handleReset(item)"
+                >
+                  Reset
+                  <v-icon small>refresh</v-icon>
+                </v-btn>
+              </div>
+
+              <JsonInput
+                ref="kvRef"
+                v-model="KvValueInput"
+                class="text-body-1 mt-2 mb-5"
+                :selected-type="
+                  selectedTypeIndex > 0
+                    ? kvTypes[selectedTypeIndex].value
+                    : kvTypes[jsonEditorType(item)].value
+                "
+                :placeholder-text="JSON.stringify(item.value)"
+                @invalid-secret="setInvalidKV"
+              >
+                <v-menu top offset-y>
+                  <template #activator="{ on }">
+                    <v-btn
+                      text
+                      small
+                      class="position-absolute"
+                      :style="{
+                        bottom: '25px',
+                        right: '140px',
+                        'z-index': 3
+                      }"
+                      color="accent"
+                      :loading="isSettingKV"
+                      :disabled="
+                        !KvValueInput ||
+                          KvValueInput == checkValue(item.value) ||
+                          invalidKV
+                      "
+                      v-on="on"
+                      @click="setKV"
+                    >
+                      Save
+                    </v-btn>
+                  </template>
+                </v-menu>
+                <v-menu top offset-y>
+                  <template #activator="{ on }">
+                    <v-btn
+                      text
+                      small
+                      class="position-absolute"
+                      :style="{
+                        bottom: '25px',
+                        right: '80px',
+                        'z-index': 3
+                      }"
+                      color="accent"
+                      v-on="on"
+                      >Type</v-btn
+                    >
+                  </template>
+                  <v-list>
+                    <v-list-item-group
+                      v-model="selectedTypeIndex"
+                      color="primary"
+                    >
+                      <v-list-item
+                        v-for="(item, index) in kvTypes"
```

Comment:
I would switch item to type so that it's got a unique name.
```suggestion
                        v-for="(type, index) in kvTypes"
```